examples/always_returning_example.py

```diff
@@ -0,0 +1,21 @@
+def loop():
+    for j in range(0, 1):
```

Comment:
@Jasminewwwu increase the loop index. The current range does indeed distract from the check.